test/runtests.jl

```diff
@@ -0,0 +1,7 @@
+using QRDecoders
+using Test
+using QRCode
+import QRCode: formatinfo
+
+include("../src/qrcoder.jl")
```

Comment:
Seems not needed? This should be reached via `using QRDecoders`.

Reply:
> Seems not needed? This should be reached via `using QRDecoders`.

Yes, the variable `formatinfo` was imported for testing purposes only (to ensure the correctness of `qrformat(fmt::Int)`). It might be better if I put it in the `runtests` file.

source code:
```jl
@testset "Format decoding" begin
    ## test format encoding
    tag = true
    for ((qua, mask), val) in formatinfo
        fmt = qrformat(quality2binary[qua] << 3 ⊻ mask)
        val = parse(Int, join(Int.(val)); base=2)
        if fmt != val
            tag = false
            break
        end
    end
    @test tag
end
```

---

btw, the table `formatinfo` don't seem necessary. It costs more to convert `BitArray` to integer in order to do XOR operation while costing little to call `qrformat(fmt)` (1.8 μs per time).
```jl
const formatinfo = Dict{Tuple{ErrCorrLevel, Int64}, BitArray{1}}(...)`
```

Reply:
> Get it. include("../src/qrcoder.jl") => include("qrcoder.jl")

I mean, since `src/QRDecoder.jl` already has `include("qrcoder.jl")`, `using QRDecoder` will already load this "qrcoder.jl" file. Why would you `include` it again explicitly?

Reply:
> the table formatinfo don't seem necessary. It costs more to convert BitArray to integer in order to do XOR operation while costing little to call qrformat(fmt) (1.8 μs per time).

If you have a good solution, feel free to open a PR in QRCode side.